packages/gestalt/src/Checkbox.js

```diff
@@ -34,20 +34,19 @@ type Props = {|
    */
   id: string,
   /**
-   * An optional Image component can be supplied to add an image to each checkbox. Spacing is already accounted for; simply specify the width and height.
+   * An optional Image can be supplied to add an image to each Checkbox. See the [with Image variant](https://gestalt.pinterest.systems/checkbox#With-Image) to learn more.
    */
   image?: Node,
   /**
-   * Used to indicate a state that is neither checked nor unchecked — e.g. a "Select all" checkbox when not all items are selected or unselected.
-   * Indeterminism is purely presentational - the value of a checkbox and its indeterminism are independent.
+   * Indicates a state that is neither checked nor unchecked. See the [state variant](https://gestalt.pinterest.systems/checkbox#State) to learn more.
    */
   indeterminate?: boolean,
   /**
-   * The label for the input. Be sure to localize the text.
+   * The label for the input. Be sure to localize the text. See the [accessibility section](https://gestalt.pinterest.systems/checkbox#Labels) to learn more.
    */
   label?: string,
   /**
-   * A unique name for the input.
+   * A unique name for the input used to reference form data after it’s submitted. If the name attribute is not specified then the data of would not be sent in the form submission.
```

Comment:
"then the date of **the checkbox** would not"

Reply:
DONE